### ISSUE TITLE
Announcement name might be confusing

### ISSUE THREAD

**Issue by buhtz** [WeblateOrg/weblate#9478]
### Describe the problem

My users often don't understand what it means that my project is "Locked". They mail me and asking.

Keep in mind that weblate to target users that don't have any technical background. They do not know about code, repositories, version control and things like that. They don't care.

### Describe the solution you'd like

Offer a way to the project owners to modify or extend the colored text box telling the users that the project is locked. I would like to extend the explanation with none technical terms, apologize and givin a date when the project will be available again.

Owners of a project are not the admins of the weblate instance.

**Comment by nijel** [comment]
There are announcements: https://docs.weblate.org/en/latest/admin/announcements.html

**Comment by buhtz** [comment]
No need to use your bot. Throw it back into its cage. I'm human and want to talk to humans not to bots.

Your answer summarize: The feature is still planed.

**Comment by buhtz** [comment]
The feature is still relevant.
I would like to customize the "Lock" message for visitors.
Announcement are not an alternative because it is push-message instead of pull.

**Comment by nijel** [comment]
Can you please clarify why announcements do not work for this?

If the lock message can be improved, let's improve it for everybody. If you lock a project for a specific reason, announcements is a good way to explain it. I don't see need for another kind of announcement specific to locking.

PS: The bot is here to close issues where we lack feedback from the original author, and we need it, so please stop ranting about it. But it was you who actually closed the issue…

**Comment by buhtz** [comment]
> Can you please clarify why announcements do not work for this?

Seems to work. Based on the features name I thought it is an email service spamming all registered translators via email.
Mhm... I can't imagine a better name as "announcements". But it is not perfect.
Maybe locate the feature where it takes effect. Make a button or something liket his right beside where the anouncments usually appear. Make it context sensitive? Here is a quick & dirty mock up

![image](https://github.com/WeblateOrg/weblate/assets/11861496/b40d701a-7afb-42cb-80da-8f8801c725ea)

> PS: The bot is here to close issues where we lack feedback from the original author, and we need it, so please stop ranting about it. But it was you who actually closed the issue…

I was not "ranting" about the bot in the last post. The problem is not that there is an automatism. The problem is that it simulates to be act like a human and being intelligent like a human. Don't let "it speak". Just use short phrases not sentences. It is not polite to users. It is the same with speech computers when you calling support hotlines. Don't make machines look like humans.

**Comment by buhtz** [comment]
What is the "Notify users" check box about?

**Comment by nijel** [comment]
> Maybe locate the feature where it takes effect. Make a button or something liket his right beside where the anouncments usually appear.

Sorry, but there can't be a button for every feature on the main page.

> What is the "Notify users" check box about?

It sends notification to subscribed users. Without that, the announcement is only displayed on the web.

**Comment by buhtz** [comment]
> > What is the "Notify users" check box about?
> 
> It sends notification to subscribed users. Without that, the announcement is only displayed on the web.

How is "subscribed" definied? Is a contributing translator subscribed by default?

Maybe modify that string to "Notify subscribed users via Email" if email is used. And add this usefull "?" behind it linking to the related entry in the manual. I often use the "?" in other parts of the weblate front end.

**Comment by nijel** [comment]
Users are automatically subscribed to projects where they contribute to, and announcements notification is turned on by default. See also https://docs.weblate.org/en/latest/user/profile.html#notifications

I've already added a help text in https://github.com/WeblateOrg/weblate/commit/9f10d5b55119c651dcb2f49607cad11f0095ade4.

As for the name, it used to be “Whiteboard”. As the best name back then, we chose “Announcements” (I think there is an issue with that discussion). I don't see “Info banner” to be a better name for this. Maybe @orangesunny or @comradekingu can comment on this as well.
